src/extension/ncwms/src/main/java/au/org/emii/geoserver/wms/Ncwms.java

```diff
@@ -48,16 +50,16 @@ public void getMetadata(HttpServletRequest request, HttpServletResponse response
             JSONObject resultJson = new JSONObject();
             resultJson.put("timesteps", urlIndexInterface.getTimesForDay(layerDescriptor, day));
             response.getOutputStream().write(resultJson.toString().getBytes());
-        }
-        else if (item != null && item.compareTo("layerDetails") == 0) {
+        } else if (item != null && item.compareTo("layerDetails") == 0) {
```

Comment:
Not that I like that convention, but it says to put `else` under the curly bracket. Might as well follow it.